src/test/java/uk/gov/dhsc/htbhf/page/BasePage.java

```diff
@@ -33,16 +37,16 @@ public void open() {
         webDriver.get(fullUrl);
     }
 
-    public WebElement findByClassName(String className) {
-        return webDriver.findElement(By.className(className));
+    public void waitForPageToLoad() {
+        wait.until(ExpectedConditions.titleIs(getPageTitle()));
     }
 
-    public WebElement findByCss(String cssSelector) {
-        return webDriver.findElement(By.cssSelector(cssSelector));
+    public List<WebElement> getRadioButtons() {
+        return webDriver.findElements(By.className("govuk-radios__item"));
     }
 
-    public void waitForPageToLoad() {
-        wait.until(ExpectedConditions.titleIs(getPageTitle()));
+    public List<WebElement> getAllRadioLabels(String option) {
+        return webDriver.findElements(By.className("govuk-label govuk-radios__label"));
```

Comment:
I don't understand what the string is doing. Is this selecting multiple classnames? or is this some sort of selector (omitting `.` for each classname)? If it is a selector, could it just be:
```
govuk-radios__label
```

Reply:
Its there by mistake, I'll remove it.